### PR TITLE
add FLAC seek & resync for older player

### PR DESCRIPTION
**Pull request by philippe44**
I've added the CRC8 in Perl directly. It's slow but only used rarely, so it should be fine. I've not tested much and it's not ready for 
release 'I know there are some style violations that I will address), but just to give you and idea if what it can be. It nicely leverage the modifications I've done on HTTP and header management, without any modification. I've just added the scan function in Remote.pm in FLAC.pm, so that's really a good "surprise"

### CODE REVIEW

**Comment by michaelherger** [comment]
Hmm... did you miss a file or something? Streaming TIDAL I don't see any of this new code being called. Tried proxied and direct streaming.

File: Slim/Formats/FLAC.pm [hunk 1]
```diff
@@ -939,6 +939,131 @@ sub scanBitrate {
...
+    0x76, 0x71, 0x78, 0x7F, 0x6A, 0x6D, 0x64, 0x63,
+    0x3E, 0x39, 0x30, 0x37, 0x22, 0x25, 0x2C, 0x2B,
+    0x06, 0x01, 0x08, 0x0F, 0x1A, 0x1D, 0x14, 0x13,
+    0xAE, 0xA9, 0xA0, 0xA7, 0xB2, 0xB5, 0xBC, 0xBB,
+    0x96, 0x91, 0x98, 0x9F, 0x8A, 0x8D, 0x84, 0x83,
+    0xDE, 0xD9, 0xD0, 0xD7, 0xC2, 0xC5, 0xCC, 0xCB,
+    0xE6, 0xE1, 0xE8, 0xEF, 0xFA, 0xFD, 0xF4, 0xF3
+);
+
+sub crc8(const void * data, size_t size) {
```
Comment (michaelherger) [inline Slim/Formats/FLAC.pm:1056]: That's not Perl, is it?...

**Comment by philippe44** [comment]
It’s the same thing as for mp4. Because you disable the scanURL, you need to manually scanner the flac url as well, that’s the reason why. I tested on a regular http server, not on tidal

**Comment by michaelherger** [comment]
Hmm... I have a few tweaks and fixes I wanted to submit a pull request for. But Github wouldn't allow me to submit a PR for a branch which has a PR on my repository?... See https://github.com/Logitech/slimserver/commit/11436f4990b3531c6c5232712340e1f63286fb23. This allows me to seek TIDAL FLAC streams, too.

File: Slim/Formats/FLAC.pm (hunk 1 above)
Comment (philippe44) [reply Slim/Formats/FLAC.pm:1056]: indeed, a leftover from the C where I took it. Funny enough, Perl does not whine

**Pull request merged**